### PR TITLE
Update current.yml

### PR DESCRIPTION
**Pull request by grtb**
add wetest.athens2023

### CODE REVIEW

**Comment by grtb** [comment]
add wetest.athens

File: _data/current.yml [hunk 1]
```diff
@@ -191,6 +191,12 @@
   url: https://starwest.techwell.com/?utm_source=testingconferences
   twitter: techwell
   status: <a href="https://starwest.techwell.com/speak-starwest" target="_blank">CFP is Open</a> until February 12, 2023
+  
+- name: WeTest.Athens 2023
+  location: Athens, Greece, Europe
+  dates: "October 5-6, 2023"
+  url: https://www.linkedin.com/company/33295477/
```
Comment (ckenst) [inline _data/current.yml:198]: Is this the best url to use right now?

Comment (grtb) [reply _data/current.yml:198]: Yes, we will update this, once the website is up.

Comment (grtb) [reply _data/current.yml:198]: Site just came up : https://wetest-athens.gr/

**Review by ckenst — APPROVED** [review]

**Pull request merged**